### DISCUSSION TITLE
Use sw8 header to propagation traceid

### DISCUSSION THREAD

**Discussion by ac3225020** [apache/skywalking#9511] [General]
i want to use skywallking to trace frontend and backend in one traceid,so i created a traceid and sw8 header , then i  send a http request to my springboot backend with  sw8 header. The backend receive this header, but no use it,and generate a new traceid。
Is my thought correct? or how can i do this with skywalking?

**Comment by wu-sheng** [comment]
If you follow the protocol, the agent will. There is no difference between native agent or your own codes.
